scribe-tool/scrawl.js

```diff
@@ -377,15 +377,19 @@
 
           // try to find the person by full name, last name, and then first name
           for(var i = 0; i < people.length; i++) {
-            var person = people[i].replace(/^\s/, '').replace(/\s$/, '');
-            var lastName = person.split(' ')[1];
-            var firstName = person.split(' ')[0];
-            if(person in aliases) {
-              scrawl.present(context, aliases[person]);
-            } else if(lastName in aliases) {
-              scrawl.present(context, aliases[lastName]);
+            if (!people[i]) {
```

Comment:
No idea if this works... assuming it does, I'm not going to try to figure it out... we'll find out if it doesn't in time. LGTM.

Reply:
i'm using javascript's generous if on strings -- this matches is the string is whitespace. I tested this and it worked for our folks who typed `present+` instead of `present+ self`